python/lsst/daf/butler/remote_butler/_registry.py

```diff
@@ -271,7 +281,15 @@ def queryCollections(
         flattenChains: bool = False,
         includeChains: bool | None = None,
     ) -> Sequence[str]:
-        raise NotImplementedError()
+        if includeChains is None:
```

Comment:
@TallJimbo what's the plan for the `datasetType` parameter? It's not in this implementation and is ignored in sql registry, but should the client code forward it to the server on the assumption that at some point the direct butler might use it? Or are we okay acting like it doesn't exist (the documentation doesn't seem to say that it's ignored and sql_registry mentions DM-24939 which is Done but hasn't fixed the problem).

Reply:
If it gets implemented in DirectButler someday it can get added here at the same time.  It sounds like we're leaning towards re-designing this interface entirely, so we can drop it from the new interface or make it actually work there.

I don't like having unused variables/features hanging around because their existence implies to future readers that they are used for something, which can end up wasting a lot of time.

Reply:
I think we drop it here, and RFC dropping it from DirectButler (and the CLI) when we RFC dropping regex support there.